dot/import.go

```diff
@@ -21,7 +21,9 @@ import (
 )
 
 // ImportState imports the state in the given files to the database with the given path.
-func ImportState(basepath, stateFP, headerFP string, stateTrieVersion trie.TrieLayout, firstSlot uint64) error {
+func ImportState(
+	basepath, stateFP, headerFP string, stateTrieVersion trie.TrieLayout,
+	genesisBABEConfig *types.BabeConfiguration, firstSlot uint64) error {
```

Comment:
```suggestion
func ImportState(basepath, stateFP, headerFP string, stateTrieVersion trie.TrieLayout,
	genesisBABEConfig *types.BabeConfiguration, firstSlot uint64) error {
```

Common practice is sorta like this